### ISSUE TITLE
Ability to provide files as previews instead of fids 

### ISSUE THREAD

**Issue by rodrigoaguilera** [drupal-fancy/view_mode_selector#11]
Great module!!

The forms for the preview stores a fid in the settings array.
When I deploy that to another server with features the fid is not the same.

Right now I'm solving that with a form alter wherever we show the previews but adding the ability to have a fixed path may be nice.

Thanks.

**Comment by yannickoo** [comment]
Okay, I see the problem here and will work on this later this week :)

**Comment by yannickoo** [comment]
Yeah we should support file paths as well so that the icons can survive deployments.
